tests/test_bibstruct.py

```diff
@@ -32,6 +32,46 @@ def test_simple(self):
         key = bibstruct.generate_citekey(bibentry)
         self.assertEqual(key, 'Salinger1961')
 
+    def test_no_modifier(self):
```

Comment:
Those tests are good. We need, as mentioned above, some usecase tests with tricky cases to ensure the commands work and fail gracefully.